Content.Shared/GameObjects/Components/Mobs/SharedHumanoidAppearanceComponent.cs

```diff
@@ -41,6 +41,7 @@ public virtual Sex Sex
         {
             Sex.Female => Gender.Female,
             Sex.Male => Gender.Male,
+            Sex.Classified => Gender.Neuter,
```

Comment:
Why is this using `it/its`? Pronouns for not disclosed or unknown gender has been established as `they/them` for quite a while. As in, at least 14th century.